bouter/free/__init__.py

```diff
@@ -5,39 +5,13 @@
 
 class FreelySwimmingExperiment(Experiment):
 
-    def get_n_segments(self, prefix=True):
-        if prefix:
-
-            def _tail_part(s):
-                ps = s.split("_")
-                if len(ps) == 3:
-                    return ps[2]
-                else:
-                    return 0
-
-        else:
 
-            def _tail_part(s):
-                ps = s.split("_")
-                if len(ps) == 2:
-                    return ps[1]
-                else:
-                    return 0
-
-        tpfn = np.vectorize(_tail_part, otypes=[int])
-        return np.max(tpfn(self.behavior_log.columns.values)) + 1
+    def get_n_segments(self, prefix=True):
```

Comment:
It should be consistent with the embedded fish, like this:
https://github.com/portugueslab/bouter/blob/7c920588589166ffa222c5260e1a45ff420f880d/bouter/embedded/__init__.py#L20-L23